test/babel-eslint.js

```diff
@@ -298,6 +298,16 @@ describe("babylon-to-espree", () => {
     assert.strictEqual(babylonAST.tokens[1].type, "Punctuator");
   });
 
+  // Espree doesn't support Flow enums yet
```

Comment:
It's unlikely Espree will ever support Flow (there are no plans to do so at the moment, at least).

```suggestion
  // Espree doesn't support Flow enums
```